fake-ipa/fake_ipa/main.py

```diff
@@ -292,7 +292,13 @@ def main():
         'FAKE_IPA_ADVERTISE_ADDRESS_IP: %s',
         app.config.get('FAKE_IPA_ADVERTISE_ADDRESS_IP')
         )
-    app.run(host=app.config.get('SUSHY_FAKE_IPA_LISTEN_IP', '0.0.0.0'),
+
+    if  app.config.get("FAKE_IPA_CERTFILE") and app.config.get("FAKE_IPA_KEYFILE"):
```

Comment:
Same here. Also IMO it's better to have these as variables, so that we don't repeat `app.config.get()` too many times.

Reply:
done thanks